packages/reports/src/Constants/data-table-constants.tsx

```diff
@@ -67,15 +91,15 @@ export const getStatementTableColumnsTemplate = currency => [
     {
         title: localize('Transaction time'),
         col_index: 'date',
-        renderCellContent: ({ cell_value }) => {
+        renderCellContent: ({ cell_value }: TCellContentProps) => {
             return <span>{cell_value} GMT</span>;
         },
     },
     {
         key: 'mode',
         title: localize('Transaction'),
         col_index: 'action_type',
-        renderCellContent: ({ cell_value, passthrough, row_obj }) => (
+        renderCellContent: ({ cell_value, passthrough, row_obj }: TCellContentProps) => (
```

Comment:
```suggestion
        renderCellContent: ({ cell_value, passthrough, row_obj }) => (
```
We don’t need to declare the type for each `renderCellContent` callback as TS can infer it, So maybe we can remove them? 🤔